docs/audits/index.md

```diff
@@ -0,0 +1,24 @@
+# Jupyter Accessibility Audits
```

Comment:
If I were someone coming to this page for the first time, I might wonder why some of the audits are in GitHub issues, while two are in these docs. Is there some way we can address that? Or maybe put all the audit links in one list, rather than splitting them into two lists?

I'm not sure what the best approach is, but currently this looks confusing to me.

Reply:
Agree - the current structure (two lists, some with in-depth content other without) make this confusing at the least, and signals some sort of "hierarchy" / "importance at worst"

Reply:
Should I only add the links to the github issues/PR?

Still we need to add the new two pages to a toctree, if not Sphinx will not include them and will not be reachable.

Reply:
Edit: I removed the single audit pages and just left the list of all the audits